### PR TITLE
add cluster list cmd to docs

### PR DESCRIPTION
**Pull request by BelSasha**

### CODE REVIEW

**Comment by BelSasha** [comment]
* **#1298** <a href="https://app.graphite.dev/github/pr/run-house/runhouse/1298?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @BelSasha and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: docs/api/cli.rst [hunk 1]
```diff
@@ -11,6 +11,6 @@ The commands can be run like follows:
    $ runhouse ssh cluster_name
 
 .. automodule:: runhouse.main
-   :members: login, logout, ssh, notebook, start, restart, status
+   :members: login, logout, ssh, notebook, start, restart, status, cluster_list
```
Comment (jlewitt1) [inline docs/api/cli.rst:14]: what is this supposed to enable? i dont see `cluster`, `cluster_list` or `list` as a command option when running `runhouse --help`

Comment (BelSasha) [reply docs/api/cli.rst:14]: image
image

File: runhouse/main.py [hunk 2]
```diff
@@ -613,7 +613,20 @@ def cluster_list(
         help="Cluster status to filter on. Supported filter values: running, terminated (cluster is not live), down (Runhouse server is down, but the cluster might be live).",
     ),
 ):
-    """Load Runhouse clusters"""
+    """
+    Load Runhouse clusters
```
Comment (jlewitt1) [inline runhouse/main.py:617]: formatting looks a little off:

**Review by jlewitt1 — APPROVED** [review]

**Pull request merged**